docs/limitations/nested-objects.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[nested-objects]]
+== Limitation with nested objects
```

Comment:
No need to repeat "limitation" in the heading. I'd be inclined to explicitly state the problem. "Aggregations with Nested Objects not Supported"